app/components/StartServer/DeletePresetButton.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component, PropTypes } from 'react';
+import { Button } from 'react-photonkit';
+
+import styles from './StartButton.css';
+
+export default class DeletePresetButton extends Component {
+  static propTypes = {
+    presetDeleting: PropTypes.bool.isRequired,
+    deletePreset: PropTypes.func.isRequired,
+  }
+
+  render () {
+    const {presetDeleting, deletePreset} = this.props;
+
+    return (
+      <div>
+        <Button className={styles.startButton} type="button"
+         ptStyle={presetDeleting ? "disabled" : "warning"}
+         text={presetDeleting ? "Deleting..." : "Delete Preset"}
+         onClick={deletePreset}
+        />
+        <input type="submit" hidden={true} />
```

Comment:
It allows this to be part of a form and receive form submit events via the enter key. But I think it probably doesn't make sense to have on this particular button since it's not part of a form. Will remove.